api/service/dashboard/dashboard.go

```diff
@@ -65,7 +65,15 @@ func (d *dashboard) Aprs(addr ...Addr) (Aprs, error) {
 }
 
 // Pools implements Dashboard.
-func (d *dashboard) Pools() (Pools, error) {
+func (d *dashboard) Pools(tokens ...Addr) (Pools, error) {
+	var tokensCond string
+	if len(tokens) > 0 {
+		var tokensStr string
+		for _, t := range tokens {
+			tokensStr += string(t)
+		}
+		tokensCond = " and (p.asset0 in ('" + tokensStr + "') or p.asset1 in ('" + tokensStr + "'))"
+	}
```

Comment:
risk of SQL injection

Reply:
[b5e3f36](https://github.com/dezswap/dezswap-api/pull/14/commits/b5e3f368a091fee146e73c641bac7917b195159b)